update-keys.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh -ex
 
-curl -fsSLo- --compressed https://github.com/nodejs/node/raw/master/README.md | awk '/^gpg --keyserver hkps:\/\/keys\.openpgp\.org --recv-keys/ {print $NF}' > keys/node.keys
+curl -fsSLo- --compressed https://github.com/nodejs/node/raw/main/README.md | awk '/--recv-keys.*#/{ gsub(/^.*--recv-keys\s+/,"");gsub(/\s+#.*$/,""); print }' > keys/node.keys
```

Comment:
this has always felt brittle - thoughts on hosting some file on e.g. nodejs.org that has this info rather than getting it from the readme?

Reply:
The release WG maintains https://github.com/nodejs/release-keys/, for which there is https://github.com/nodejs/release-keys/blob/main/keys.list although that contains all keys (and not just the current active set).